Makefile

```diff
@@ -264,16 +264,32 @@ format: pre-format
 pre-clippy: unset-override
 	@rustup component add clippy
 
+ALLOWED_CLIPPY_LINTS=-A clippy::module_inception -A clippy::needless_pass_by_value -A clippy::cognitive_complexity \
+	-A clippy::unreadable_literal -A clippy::should_implement_trait -A clippy::verbose_bit_mask \
+	-A clippy::implicit_hasher -A clippy::large_enum_variant -A clippy::new_without_default \
+	-A clippy::neg_cmp_op_on_partial_ord -A clippy::too_many_arguments \
+	-A clippy::excessive_precision -A clippy::collapsible_if -A clippy::blacklisted_name \
+	-A clippy::needless_range_loop -A clippy::redundant_closure \
+	-A clippy::match_wild_err_arm -A clippy::blacklisted_name -A clippy::redundant_closure_call \
+	-A clippy::identity_conversion -A clippy::new_ret_no_self
+
+# PROST feature works differently in test cdc and backup package, they need to be checked under their folders.
```

Comment:
What's the difference?

Reply:
tests, cdc and backup are not included in tikv's "prost-codec", so `cargo clippy --feature "prost-codec"` does not work. We have to cd their folder and then run the command.

Reply:
Can they be solved in other way? It certainly contributes to compile time to recompile several times.